Reference/Mapping/index.md

```diff
@@ -0,0 +1,190 @@
+---
+versionFrom: 8.1.0
+---
+
+# UmbracoMapper
+
+:::tip
+UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, i.e. mappings need to be hand-written.
+
+This is not to be confused with the [package by Andy Butland](https://our.umbraco.com/packages/developer-tools/umbraco-mapper)
```

Comment:
confused with the [V7 UmbracoMapper package by Andy Butland] of the same name.

Reply:
Andy's "Umbraco Mapper" [also supports v8 too](https://web-matters.blogspot.com/2019/03/umbraco-mapper-new-releases-supporting-v8.html), _just to add to the confusion._